ingestion/tests/integration/ometa/test_ometa_es_api.py

```diff
@@ -99,6 +100,9 @@ def setUpClass(cls) -> None:
 
         cls.entity = cls.metadata.create_or_update(create)
 
+        # Leave some time for indexes to get updated, otherwise this happens to fast
+        time.sleep(5)
```

Comment:
isn't there a way to wait until the indexes are updated? I think it would be a cleaner solution.

Reply:
hehe I was lazy. I could wait until I can fetch the table from the index. Let me do that